.github/workflows/deploy.yml

```
@@ -0,0 +1,126 @@
+name: Deploy
+on:
+  release:
+    types: [published]
+
+env:
+  CIBW_SKIP: "cp27-* cp34-* cp35-* *i686 pp* *win32"
+
+  # Linux specific build settings
+
+  CIBW_BEFORE_ALL_LINUX: |
+    curl -OsL https://bitbucket.org/eigen/eigen/get/3.3.7.tar.gz
+    tar xzf 3.3.7.tar.gz eigen-eigen-323c052e1731/Eigen --strip-components 1
+    tar xzf 3.3.7.tar.gz eigen-eigen-323c052e1731/unsupported --strip-components 1

```

Need to apply changes as per #44
```suggestion
    curl -OsL https://gitlab.com/libeigen/eigen/-/archive/3.3.7/eigen-3.3.7.tar.gz
    tar xzf eigen-3.3.7.tar.gz eigen-3.3.7/Eigen --strip-components 1
    tar xzf eigen-3.3.7.tar.gz eigen-3.3.7/unsupported --strip-components 1

```